commands/management/add_mysql.go

```diff
@@ -202,6 +220,7 @@ func (cmd *addMySQLCommand) Run() (commands.Result, error) {
 			Username:       cmd.Username,
 			Password:       cmd.Password,
 			AgentPassword:  cmd.AgentPassword,
+			Credentials:    cmd.CredentialsFile,
```

Comment:
I think we should override Username, Password and AgentPassword on client side, so we don't need to pass them to Server as new field